### DISCUSSION TITLE
Welcome to pronostics-jdl Discussions!

### DISCUSSION THREAD

**Discussion by BMunier** [BMunier/pronostics-jdl#60] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lelie21** [comment]
Hello, my name is Brian, I live in the kitchen.

**Reply by BMunier** [reply]
Hola Brian!

**Reply by lelie21** [reply]
goeiedag
